subsys/bluetooth/services/fast_pair/fmdn/state.c

```diff
@@ -580,7 +580,7 @@ static int fmdn_tx_power_set(uint16_t handle, int8_t *tx_power)
 		return -EINVAL;
 	}
 
-	buf = bt_hci_cmd_create(BT_HCI_OP_VS_WRITE_TX_POWER_LEVEL, sizeof(*cp));
```

Comment:
Should we also update comments here https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/bluetooth/controller/hci_internal_wrappers.c#L31-L38 ?